### ISSUE TITLE
[en] learn vs study

### ISSUE THREAD

**Issue by kostyfisik** [languagetool-org/languagetool#571]
probably another pair for ngram:
WRONG To read, memorize facts, attend school, etc., in order to study about a subject
CORRECT To read, memorize facts, attend school, etc., in order to learn about a subject

The first sentence is not marked in the current online LT.

**Comment by danielnaber** [comment]
Seems to work, 38.8% of wrong uses are detected. That is, are these always different or are these cases where you can use either one? A native speaker should comment on that, maybe @MikeUnwalla?

`learn; study; 10000000;                                                                            # p=0.997, r=0.388, 981+999, 3grams, 2016-10-08`

**Comment by kostyfisik** [comment]
I am not a native English speaker, but the difference seems to be quite easy - to study is more about the process, and to learn is more on the result. E.g. I studied philosophy in the university, but learned nothing about it :)

**Comment by MikeUnwalla** [comment]
I agree with @kostyfisik. People go school/college/university to learn. They study in order to learn. Oxford Collocations Dictionary for students of English has these definitions of 'study':

1. Spend time learning about something.
2. Examine something carefully.

The meanings of study/learn are not interchangeable, but the following pair of sentences shows that for a given grammar structure, both verbs are possible:

- The researchers learnt the causes of the problem.
- The researchers studied the causes of the problem.

**Comment by kostyfisik** [comment]
Is it possible to LT just to warn is such places with some extended description? So in 38.8% it should provide a possible correction. If it is ok - do not show anything. In the case of possible confusion - just warn with the detailed description of the difference between learn and study. It is due to the fact that while both sentences from @MikeUnwalla are correct from the grammatical point of view,  they still have quite a different meaning, as soon as if researchers learnt the causes - they obtained some new knowledge. On the other side, if the researchers studied the causes - it only means that they spend tax-payers money... 

BTW, my first example should probably be treated as a confusion.  It is quite legal, if you go to school in order to study whatever, just because otherwise you parents will cut your pocket expenses, without any motivation to learn something new and useful....  such a sad story :( ...
